go/fn/object.go

```diff
@@ -633,6 +633,15 @@ func (o *KubeObject) IsLocalConfig() bool {
 	return true
 }
 
+// IsLocalConfig is a function that can be used by `Where` to filter out local config
```

Comment:
nit: IsLocalConfig determines whether a KubeObject (or KRM resource) has the `config.kubernetes.io/local-config: true`  annotation.  https://kpt.dev/reference/annotations/local-config